envs/r-caret.yaml

```diff
@@ -1,10 +1,10 @@
 name: r-caret
-channels:
-  - bioconda
-  - conda-forge
-  - defaults
+#channels:
```

Comment:
This channel order was giving me issues with conflicting dependencies, but my system defaults were fine. You should review these.